### DISCUSSION TITLE
Laggy/flashy loading of blocks

### DISCUSSION THREAD

**Discussion by tryashtar** [Mojang/minecraft-editor#36] [Bugs]
When clicking the "block type" option, a laggy animation starts that populates the entries and resizes the vertical scroll. Scrolling it is also laggy.

https://user-images.githubusercontent.com/26075577/223856586-c69662cc-4a0e-4553-8de4-e385d758f307.mp4

Expected results: it should be responsive and load and scroll instantly.

**Comment by minecraftchloe** [comment]
Looks like in this one, too, your FPS is really low - what other processes do you have running that is causing your FPS to tank?

**Reply by tryashtar** [reply]
I apologize, I think I was a bit hasty in reporting these performance symptoms separately.

That being said, this is definitely worse performance than I would expect with my specs. As you can see in the recording, Minecraft is the only application open. I don't have anything running in the background that isn't stock Windows.
